src/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -457,9 +458,9 @@ by the data capture apps to identify this program stage.
  | **Hide due dates** | Select checkbox to hide due dates for events. |
  | **Feature type** |  Sets whether the program is going to capture a geographical  feature type or not.  <br>  * **None:** Nothing is captured.<br> * **Polygon:** An area is captured. For single  event programs the area will be the area representing the  event being captured. For tracker programs, the area will  represent the area of the enrollment. <br> * **Point:** A point/coordinate is captured. For  single event programs the point will be representing the  event being captured. For tracker programs, the point will  represent the enrollment. |
  | **Pre-generate event UID** | Select check box to pre-generate unique event id numbers. |
- | **Description of report date** | Type a description of the report date.<br>  <br>This description is displayed in the data entry form. |
- | **Description of due date** | Type a description of the due date. |
- | **Referral*** | Flag to indicate if program stage is referral or not. |
+ | **Custom label for report date** | Type a description of the report date.<br>  <br>This description is displayed in the data entry form. |
+ | **Custom label for due date** | Type a description of the due date. |
+ | **Custom label for:**<br> - **program stage** <br> - **event** | These custom labels will, on a program-specific level, replace these terms in certain DHIS2 apps. It is important to note that this configuration does not distinguish between singular and plural, so the label should consider this. These labels are currently only consumed in the DHIS2 Android app. |
```

Comment:
Same as above, I suggest rewording the last sentence to "Currently, custom labels are only used by the DHIS2 Android app."